src/Bridge/Symfony/DependencyInjection/Compiler/TranslationPass.php

```diff
@@ -25,7 +25,8 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        $translator = $container->findDefinition('translator.default');
+        $translator = $container->findDefinition('translator.default')
+            ->setPublic(true);
```

Comment:
We should not make a vendor service public our-self. Why do you need this?

Reply:
Public should be, because in sf 4 all services are private by [default](https://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default). And if i am not making it public i think we would not be able to use it on without making it public here or declaring it in services.yaml file as public.

Reply:
Ok, maybe you are right, but when i am running tests, i get this:
![image](https://user-images.githubusercontent.com/2676279/40585237-cfeaf8f8-61b7-11e8-88e2-99da6fb79ccb.png)
Maybe i don't understand something :)

Reply:
Ok, i think the problem is here in test, in test we call the service to test it.

Reply:
It's a know problem fixed with Symfony 4.1: https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing

For the other versions, the simpler way is to make a public alias AFAIK.